data/ecosystems/m/multiversx-elrond.toml

```diff
@@ -2001,3 +2001,198 @@ url = "https://github.com/zdenal/elrond-amm"
 
 [[repo]]
 url = "https://github.com/ziestwilky/elrond-smartcontract"
+
+[[repo]]
+url = "https://github.com/AIRO-NETWORK/airo-core-backend"
+
+[[repo]]
+url = "https://github.com/jimcoco/Blend-MultiversX"
+
+[[repo]]
+url = "https://github.com/combased-dev/calileo-frontend"
+
+[[repo]]
+url = "https://github.com/raresraulea/sw-licensing-api"
+
+[[repo]]
+url = "https://github.com/CheckerChain"
+
+[[repo]]
+url = "https://github.com/Chessout/chessout"
+
+[[repo]]
+url = "https://github.com/CoinDrip-finance/protocol-v2"
+
+[[repo]]
+url = "https://github.com/CosmicF?tab=repositories"
```

Comment:
This is an org link, not a repo link. Please clean and add this org link to the org list section at the top of the file if you consider that the entire org is building / built in MultiversX.

Reply:
They've hidden their repos in the meantime.